artifacts/OWNERS

```diff
@@ -1,4 +1,14 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
+options:
+  no_parent_owners: true
```

Comment:
Why? Preferably as a comment in this file

Reply:
Added:

```markdown
  # Root approvers for this repo include contributors who are not involved
  # in artifact promotion. We're blocking root approvals here to ensure
  # manifests are reviewed by Release Engineering and/or WG K8s Infra
  # approvers.
  #
  # Long-term, a separate repo has been proposed for these artifact promotion
  # manifests: https://github.com/kubernetes/org/issues/2342
```